packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -918,9 +923,14 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     async accountSwitcherListener() {
+        if (this.root_store.client.landing_company_shortcode === 'maltainvest') {
+            const { active_symbols } = await WS.authorized.activeSymbols();
+            if (!active_symbols || !active_symbols.length) {
+                showDigitalOptionsUnavailableError(this.root_store.common.showError);
```

Comment:
isn't this handled in `setActiveSymbols` above already?

Reply:
No. We have two scenarios here. One is when we have an error when switching accounts and the other without errors. In the first case `setActiveSymbols` will run and for the second this method is invoked.